aws/eks-helm/metrics-server/main.tf

```diff
@@ -8,8 +8,36 @@ resource "helm_release" "metrics_server" {
   keyring    = ""
 
   # If true, allow unauthenticated access to /metrics.
-  set {
-    name  = "metrics.enabled"
-    value = false
+  dynamic "set" {
+    for_each = var.set_metrics_enabled == null ? [] : [var.set_metrics_enabled]
```

Comment:
can we group the 3 sets with something like
```tf
for_each = [for v in [var.set_metrics_enabed, var.set_container_port, var.set_rbac_create]: v if v != null]
```

Reply:
I got it, thank you.